notebooks/spa/_toctree.yml

```diff
@@ -0,0 +1,37 @@
+- title: Open-Source AI Cookbook
```

Comment:
New sections and subsections have been introduced so we may need to add them here as well